src/CMakeLists.txt

```diff
@@ -9,7 +9,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 add_subdirectory(pybind11 cmake_pybind11)
 
 # set up Arrow
-find_package(Arrow REQUIRED)
+execute_process(
```

Comment:
documented here: https://arrow.apache.org/docs/python/extending.html#building-extensions-against-pypi-wheels

lol i really need to read the docs better